embark/uploader/test_unpacker.py

```diff
@@ -0,0 +1,57 @@
+import os
+import shutil
+
+from django.test import TestCase
+
+from .unpacker import unpacker
+import unittest as ut
+
+
+class unpacker_test(TestCase):
+
+    def setUp(self):
+        self.unpacker = unpacker
+
+    def test_supported_formats(self):
```

Comment:
don't forget to comment:)

Reply:
Will do next time.  :)
Tried to use mnemonic names.